src/environments/environment.ts

```diff
@@ -3,9 +3,20 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: true
+  production: false,
+  firebase: {
```

Comment:
You can also import the firebase from your `my-env.ts`, there is no sense to do it in only one file.
The goal here was to not push your firebase environment, fail x)

Reply:
yes xD